app/javascript/components/TeacherSetDetails.jsx

```diff
@@ -644,17 +644,19 @@ export default function TeacherSetDetails(props) {
         paddingInlineStart="m"
         type="ul"
       >
-        {books.map((book, i) => (
-          <li key={"ts-books-key-" + i}>
-            <Link 
-              id={"ts-books-" + i}
-              onClick={(e) => {
-                windowScrollTop(e, book);
-              }}>
-              {book.title}
-            </Link>
-          </li>
-        ))}
+        { books.map((book, i) => (
```

Comment:
If there are zero book titles for a teacher set, show `n/a` for the `Book titles` row.  For example, when the teacher set is a game.